docs/scim/v1/scimv1.md

```diff
@@ -0,0 +1,361 @@
+# SCIM Support in V1
```

Comment:
I wouldn't reference "v1" since that's internal naming. Maybe something like "SCIM Guide", "Guide to using SCIM",  "Using SCIM", or just "SCIM".